Mathlib/SetTheory/Surreal/Multiplication.lean

```diff
@@ -210,24 +210,24 @@ lemma ih1_neg_right : ih1 x y → ih1 x (-y) :=
 
 variable (ih : ∀ a, ArgsRel a (Args.P1 x y) → P124 a)
 
-lemma ihnx (h : IsOption x' x) : (x' * y).Numeric :=
+lemma left_numeric_of_ih (h : IsOption x' x) : (x' * y).Numeric :=
   ih (Args.P1 x' y) $ TransGen.single $ cutExpand_pair_left h
 
-lemma ihny (h : IsOption y' y) : (x * y').Numeric :=
+lemma right_numeric_of_ih (h : IsOption y' y) : (x * y').Numeric :=
   ih (Args.P1 x y') $ TransGen.single $ cutExpand_pair_right h
 
-lemma ihnxy (hx : IsOption x' x) (hy : IsOption y' y) : (x' * y').Numeric :=
+lemma left_right_numeric_of_ih (hx : IsOption x' x) (hy : IsOption y' y) : (x' * y').Numeric :=
   ih (Args.P1 x' y') $ TransGen.tail (TransGen.single $ cutExpand_pair_right hy) $
     cutExpand_pair_left hx
 
-lemma ih1xy : ih1 x y := by
+lemma ih1_of_ih : ih1 x y := by
   rintro x₁ x₂ y' h₁ h₂ (rfl|hy) <;> apply ih (Args.P24 _ _ _)
   swap
   refine TransGen.tail ?_ (cutExpand_pair_right hy)
   all_goals { exact TransGen.single (cutExpand_double_left h₁ h₂) }
 
-lemma ih1yx : ih1 y x :=
-  ih1xy $ by
+lemma ih1_rev_of_ih : ih1 y x :=
+  ih1_of_ih $ by
```

Comment:
I renamed `ih1` to `IH1` according to naming convention, and then I can rename `ih1xy` to `ih1`, and `ih1yx` to `ih1_swap`. (I think `swap` is more common than `rev`, which is applied to lists/strings where there may be more than one element.)